x/uibc/quota/keeper/keeper.go

```diff
@@ -0,0 +1,56 @@
+package keeper
+
+import (
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/ibc-go/v5/modules/apps/29-fee/types"
+	porttypes "github.com/cosmos/ibc-go/v5/modules/core/05-port/types"
+
+	"github.com/umee-network/umee/v4/x/uibc"
+)
+
+type Keeper struct {
+	storeKey       storetypes.StoreKey
+	cdc            codec.BinaryCodec
+	leverageKeeper uibc.LeverageKeeper
+	ics4Wrapper    porttypes.ICS4Wrapper
+}
```

Comment:
would be great to use a concept of a keeper I presented with AvgKeeper. But we can do it later.

Reply:
ok